packages/iTwinUI-react/src/core/Table/columns/actionColumn.tsx

```diff
@@ -6,15 +6,23 @@ import React from 'react';
 import { HeaderProps } from 'react-table';
 import { Checkbox } from '../../Checkbox';
 import SvgColumnManager from '@itwin/itwinui-icons-react/cjs/icons/ColumnManager';
-import { DropdownMenu } from '../../DropdownMenu';
+import { DropdownMenu, DropdownMenuProps } from '../../DropdownMenu';
 import { IconButton } from '../../Buttons/IconButton';
 import { MenuItem } from '../../Menu';
 import { tableResizeStartAction } from '../Table';
 import { SELECTION_CELL_ID } from './selectionColumn';
 import { EXPANDER_CELL_ID } from './expanderColumn';
+import cx from 'classnames';
 
 const ACTION_CELL_ID = 'iui-table-action';
 
+type ActionColumnProps = {
+  [k: string]: unknown;
```

Comment:
Sure needed? you are not using it anywhere

Reply:
I was just experimenting with putting `[k: string]: unknown;` in every hierarchy level of the `ActionColumnProp` to try to fix [this comment thread](https://github.com/iTwin/iTwinUI-react/pull/756#discussion_r941024460). But unfortunately, wherever I tried to put a data parameter (`data-abc: 'value'`), it did not appear in the tag of the `DropdownMenu`/`ActionColumn` (as suggested [here](https://github.com/iTwin/iTwinUI-react/pull/756#discussion_r940414448)). When I figure out the right syntax, I will remove it from here and put it in the right place.

Reply:
**[DONE]** Since we don't need to test with data attributes that were suggested [here](https://github.com/iTwin/iTwinUI-react/pull/756#discussion_r940125334), we don't need to allow other props as of now, and so don't need `[k: string]: unknown;`